### ISSUE TITLE
Hulkmaka

### ISSUE THREAD

**Issue by oneliokoloa-ctrl** [w3c/wai-website#1516]
### Resource URL

https://www.w3.org/WAI/WCAG2AA-Conformance

### Description

Looking for my missing super money 

### Resource Shortname

wai-wcag-intro
